ledger-core/v2/application/testwalletapi/handlers.go

```diff
@@ -175,7 +175,50 @@ func (s *TestWalletServer) GetBalance(w http.ResponseWriter, req *http.Request)
 		TraceID: traceID,
 		Error:   "",
 	}
-	defer func() { s.mustWriteResult(w, result) }()
+
+	defer func() {
+		if len(result.Error) != 0 {
+			logger.Error(result.Error)
```

Comment:
I think we must return her and don't write `result`.

Reply:
why? Here we try to log the error and send a response with an error. What's the problem?

Reply:
I think that in result could be wrong data, that is not necessary to return, but this error is a part of result, so that's ok.